### ISSUE TITLE
"overwriteOnInitialValuesChange: false" still overwrites user input on fields not present in the initialValues

### ISSUE THREAD

**Issue by yched** [erikras/redux-form#1112]
Using 5.2.5, "overwriteOnInitialValuesChange: false" does preserve user input on fields that are present in the initialValues when they change.

However, if a field is *absent* from the initialValues (`initialValues[field] === undefined`), then it does get overriden (losing user input) when the initialValues change.

In src/initializeState.js :

```js
const makeEntry = (value, previousValue, overwriteValues) => {
  return makeFieldValue(value === undefined ? {} : {
    initial: value,
    value: overwriteValues ? value : previousValue
  });
};
```
when `value` is `undefined`, the `overwriteValues` param is ignored, and the field gets back to just `{}` even if there was a `previousValue`

**Comment by yched** [comment]
Pinging https://github.com/erikras/redux-form/issues/832, that introduced overwriteOnInitialValuesChange

Also, might be connected to https://github.com/erikras/redux-form/issues/794, that talks about providing default values for the case when initialValues does not specify all the fields handled by redux-form

**Comment by erikras** [comment]
Fix published in [`v5.3.2`](https://github.com/erikras/redux-form/releases/tag/v5.3.2).

**Comment by yched** [comment]
Awesome ! Thanks folks :-)
